### ISSUE TITLE
Works on ZS-GX1 but sharp whistling

### ISSUE THREAD

**Issue by scassarGitHub** [ant-thomas/zsgx1hacks#38]
Hi,

Thanks for the hacks. That saves me money.

Works very well on 2 ZS-GX1 with latest firmware but I heard a sharp whistling on the 2 devices when booted on SDCard and not on the original firmware.

Could you help me ?

**Comment by deorder** [comment]
If you get the sharp whistling it means the 'goke-volume' binary is not run correctly.

**Comment by ttanel** [comment]
On mine faint whistling is there even if I set volume to 0 via goke_volume. I just disconnected the speaker from motherboard and now it's completely silent.

**Comment by scassarGitHub** [comment]
Oh really ! Was it difficult to disconnect ? Do you need to break some solders or were there connectors to disconnect ?

**Comment by ttanel** [comment]
Nope, it's very easy and reversible, both mic and speaker have connectors on main board.

**Comment by scassarGitHub** [comment]
Thank you very much, I will try this WE.

**Comment by ant-thomas** [comment]
The whistling is often still there, just quieter.
I've also disconnected the speaker because it serves no purpose for me.

Disconnect the attached cable
![speaker](https://user-images.githubusercontent.com/1029851/37869139-383cf252-2faa-11e8-9f87-5895148ab88e.jpg)

**Comment by reglisse69** [comment]
Thanks a lot for all your help.

Finally v0.3 (seems) to make the noise disapeared for me (tried quickly this morning).

**Comment by reglisse69** [comment]
Ok in fact sound is always there but almost inaudible for some people and depending on the noises in the house.

I will try to do the hardware method.
